docs/intro.rst

```diff
@@ -237,7 +237,7 @@ This is the current list of error and warning codes:
 | E116     | unexpected indentation (comment)                                     |
 +----------+----------------------------------------------------------------------+
 +----------+----------------------------------------------------------------------+
-| E121 (*^)| continuation line under-indented for hanging indent                  |
+| E121(\*^)| continuation line under-indented for hanging indent                  |
```

Comment:
I think we still want it to be:

```rst
| E121 (\*^) |
```

Thanks for noticing that the `*` wasn't appearing properly though.